osmaxx/excerptexport/static/excerptexport/scripts/utm_zone_filter.js

```diff
@@ -0,0 +1,45 @@
+jQuery(document).ready(function(){
+    var utm_optgroup_html_element = '#id_coordinate_reference_system optgroup[label="UTM zones"]',
+        utm_zone_optgroup_original = jQuery(utm_optgroup_html_element).clone();
+
+    window.filterUTMZones = function (leafletGeometry) {
+        var utm_zone_optgroup = jQuery(utm_optgroup_html_element);
+
+        if (utm_zone_optgroup !== null) {
+            utm_zone_optgroup.html("");
+            getUTMZones(leafletGeometry).done(function(data){
+                var srids = data["utm_zone_srids"].sort(),
+                    options_html = '';
+                srids.forEach(function(srid){
+                    var original_text = utm_zone_optgroup_original.find('option[value=' + srid + ']').text();
+                    options_html += _optionHTML(srid, original_text);
+                });
+                utm_zone_optgroup.html(options_html);
+            }).error(function(data, other, other2, other3){
+                console.log(data, other, other2, other3);
+            });
```

Comment:
`.error()` is deprecated, use `.fail(function( jqXHR, textStatus, errorThrown ) {})` or pass the function as the `error` argument to `jQuery.ajax(...)`.